spring-batch-core/src/test/java/org/springframework/batch/core/launch/support/SimpleJobOperatorTests.java

```diff
@@ -162,18 +168,23 @@ void testStartNextInstanceSunnyDay() throws Exception {
 
 	@Test
 	void testStartNewInstanceSunnyDay() throws Exception {
-		jobParameters = new JobParameters();
+		Properties parameters = new Properties();
+		parameters.setProperty("a", "a=b");
```

Comment:
This should be `parameters.setProperty("a", "b");`. I will amend that when merging the PR.

Reply:
Thank you for correcting my mistake.